src/components/Tokens/TokenTable/TokenRow.tsx

```diff
@@ -469,7 +477,7 @@ export const LoadedRow = forwardRef((props: LoadedRowProps, ref: ForwardedRef<HT
 
   // TODO: currency logo sizing mobile (32px) vs. desktop (24px)
   return (
-    <div ref={ref} data-testid={`token-table-row-${tokenName}`}>
+    <div ref={ref} data-testid={`token-table-row-${tokenName?.split(' ').join('-')}`}>
```

Comment:
instead of all this logic, can you instead use `token-table-row-${tokenSymbol}`.

Reply:
similarly, while you're in this file, `tokenName`, `tokenSymbol`, and `tokenAddress` should all be removed in favor of direct access to `token`.

Reply:
makes sense done